Source/WebCore/rendering/svg/SVGRenderSupport.cpp

```diff
@@ -678,4 +680,27 @@ void SVGRenderSupport::paintSVGClippingMask(const RenderLayerModelObject& render
 #endif // ENABLE(LAYER_BASED_SVG_ENGINE)
 }
 
+void SVGRenderSupport::paintSVGMask(const RenderLayerModelObject& renderer, PaintInfo& paintInfo, const LayoutPoint& adjustedPaintOffset)
+{
+    auto& context = paintInfo.context();
+    if (!paintInfo.shouldPaintWithinRoot(renderer) || paintInfo.phase != PaintPhase::Mask || context.paintingDisabled())
+        return;
+
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
```

Comment:
Evil boiler plate stuff, will get better with my patch to rework the SVG resource lookup.